website/build.gradle

```diff
@@ -22,6 +22,8 @@ apply plugin: "base"
 def dockerImageTag = 'beam-website'
 def dockerWorkDir = "/repo"
 def buildDir = "$project.rootDir/build/website"
+def contentBuildDir = "$buildDir/content"
+def contentRepoDir = "$project.rootDir/website/content"
```

Comment:
This branch should now *only* have generated website content. Therefore I suggest dropping the `website/content` subdirectory. Note that this will require updating the githubpubsub publishing config when we're ready to switch over.

Reply:
org.ajoberstar.git-publish has directories/files to include/exclude. Nuked the website/generated-content directory except for pydocs and javadocs.

  preserve {
    include '**'
    exclude 'website/generated-content/**'
    include 'website/generated-content/documentation/sdks/pydoc/**'
    include 'website/generated-content/documentation/sdks/javadoc/**'
  }